study.md

```diff
@@ -33,7 +39,15 @@ How does Ember use the URL to load view-state? Which layers in Ember are
 responsible for which tasks?
 
 ```md
-<!-- your answer here -->
+Ember uses the URL to determine which route to use to get to whatever routing handler
+is associated with the model that the client would like to interact with at the
+time to display whatever data in the current view-state.
```

Comment:
Yeah, but this is kind of an unclear way to say it. The Router parses your URL and decides which Route (i.e. routed view state) to load. The Route loads a template and a model, which has the data that the template needs.